### PR TITLE
docs(upload)link send-custom-data section in overview page

### PR DESCRIPTION
**Pull request by NansiYancheva**
Related to https://github.com/telerik/blazor/issues/9000

### CODE REVIEW

File: components/upload/overview.md [hunk 1]
```diff
@@ -57,11 +57,14 @@ Steps 4 and 5 are optional, but strongly recommended.
 
 * **Save** action method
     * Its argument must be `IFormFile` or `IEnumerable<IFormFile>`. The Upload always sends files one by one, but both argument types can work.
+    * Use the Upload [`OnUpload` event]({%slug upload-events%}#onupload) to [send additional custom data with the file]({%slug upload-events%}#send-custom-data-with-the-file) as argument to the **Save** Controller Method.
```
Comment (dimodi) [inline components/upload/overview.md:60]:
```suggestion
    * Use the Upload [`OnUpload` event]({%slug upload-events%}#onupload) to [send additional custom data with the file]({%slug upload-events%}#send-custom-data-with-the-file) as argument to the **Save** controller method.
```

**Review by dimodi — COMMENTED** [review]
All existing bullets relate to the controller implementation. Injecting a bullet about the .razor file breaks the line of thought. I recommend adding the new bullet as last.

File: components/upload/overview.md [hunk 2]
```diff
@@ -60,11 +60,14 @@ Steps 4 and 5 are optional, but strongly recommended.
     * The argument name (`FormData` request key) must match the Upload [`SaveField` parameter](#upload-parameters) value. By default, that is `files`.
     * The method name must match the last part of the `SaveUrl` value.
     * The correct binding source attribute for the file argument is `[FromForm]`. Normally, .NET will infer and set it automatically.
+    * Use the Upload [`OnUpload` event]({%slug upload-events%}#onupload) to [send additional custom data with the file]({%slug upload-events%}#send-custom-data-with-the-file) as argument to the **Save** controller method.
```
Comment (yordan-mitev) [inline components/upload/overview.md:63]:
```suggestion
    * Use the Upload [`OnUpload` event]({%slug upload-events%}#onupload) to [send additional custom data with the file]({%slug upload-events%}#send-custom-data-with-the-file) as an argument to the `Save` controller method.
```

File: components/upload/overview.md [hunk 3]
```diff
@@ -60,11 +60,14 @@ Steps 4 and 5 are optional, but strongly recommended.
...
+
 * **Remove** action method
     * Its argument must be `string` or `IEnumerable<string>`.
     * The argument name (`FormData` request key) must match the Upload [`RemoveField` parameter](#upload-parameters) value. By default, that is `files`.
     * The method name must match the last part of the `RemoveUrl` value.
     * The correct binding source attribute for the file name argument is `[FromForm]`. Set it explicitly if you are using an [`[ApiController]` attribute](https://learn.microsoft.com/en-gb/aspnet/core/web-api/).
+    * Use the Upload [`OnRemove` event]({%slug upload-events%}#onremove) to [send additional custom data with the file]({%slug upload-events%}#send-custom-data-with-the-file) as argument to the **Remove** controller method.
```
Comment (yordan-mitev) [inline components/upload/overview.md:70]:
```suggestion
    * Use the Upload [`OnRemove` event]({%slug upload-events%}#onremove) to [send additional custom data with the file]({%slug upload-events%}#send-custom-data-with-the-file) as an argument to the `Remove` controller method.
```

**Review by dimodi — APPROVED** [review]

**Pull request merged**